src/steps/success.ts

```diff
@@ -1,11 +1,19 @@
-import type { Context } from 'semantic-release';
+import type { BranchObject, Context } from 'semantic-release';
 import { execSync } from 'node:child_process';
 
 import prepareTags from './prepareTags';
 import type { PluginConfigType } from './types';
 
+function isPrerelease(branchObject: BranchObject) {
+  return branchObject.prerelease !== undefined && branchObject.prerelease !== false
+}
+
 const success = (pluginConfig: PluginConfigType, context: Context) => {
-  const { options, nextRelease, logger } = context;
+  const { options, nextRelease, logger, branch } = context;
+  if (isPrerelease(branch)) {
+    logger.debug(`Not publishing any tags on a prerelease!`);
+    return;
+  }
```

Comment:
I wonder if we should add a `pluginConfig to publish major tags on prereleases also (default to `false`) if somebody would like to do this, what do you think? 🤔

Reply:
I think it is outside the scope of this PR :smiley: